agent/taskresource/credentialspec/credentialspec_windows_test.go

```diff
@@ -647,11 +644,9 @@ func TestCreateS3(t *testing.T) {
 		mockIO.EXPECT().TempFile(gomock.Any(), gomock.Any()).Return(mockFile, nil),
 		mockS3Client.EXPECT().DownloadWithContext(gomock.Any(), mockFile, gomock.Any()).Return(int64(0), nil),
 		mockFile.EXPECT().Write(gomock.Any()).AnyTimes(),
```

Comment:
Same as here. Anytimes() is not really necessary.

Reply:
done